<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommAudio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="function_stack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="client_streamAudio">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Stream Audio</string>
      </property>
      <widget class="QLabel" name="label_stream_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Multicast IP:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_stream_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>39</width>
         <height>11</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stream_ip">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="stream_port">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stream_connect">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stream_disconnect">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_stream_volume">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
      <widget class="QSlider" name="slider_stream_volume">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sliderPosition">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_stream_vol_value">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="client_playFile">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>391</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Play File</string>
      </property>
      <widget class="QLabel" name="label_file_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>39</width>
         <height>11</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_file_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Server IP:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="file_port">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="file_ip">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_upload_file">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_download_file">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Download File</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="file_name">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_file_volume">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
      <widget class="QSlider" name="slider_file_volume">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sliderPosition">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_file_vol_value">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Play a File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_resume">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stop">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_pause">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="client_microphone">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Two-Way Microphone</string>
      </property>
      <widget class="QLabel" name="label_mic_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Peer IP:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mic_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>39</width>
         <height>11</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mic_port">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="mic_ip">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_mic_connect">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_mic_disconnect">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QSlider" name="slider_mic_volume">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sliderPosition">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_mic_volume">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mic_vol_value">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="server_streamAudio">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Stream Audio</string>
      </property>
      <widget class="QPushButton" name="server_stream_start">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Server</string>
       </property>
      </widget>
      <widget class="QPushButton" name="server_stream_stop">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Server</string>
       </property>
      </widget>
      <widget class="QPushButton" name="server_stream_file">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Stream File</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Multicast Address</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Multicast Port</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="select_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="server_playFile">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Play File</string>
      </property>
      <widget class="QPushButton" name="server_file_start">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="server_file_stop">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>220</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBehaviour">
    <property name="title">
     <string>Behaviour</string>
    </property>
    <addaction name="actionClient"/>
    <addaction name="actionServer"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Function</string>
    </property>
    <addaction name="actionStream_Audio"/>
    <addaction name="actionPlay_File"/>
    <addaction name="actionMicrophone"/>
    <addaction name="actionServer_Start"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuBehaviour"/>
   <addaction name="menuFunction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionStream_Audio">
   <property name="text">
    <string>Stream Audio</string>
   </property>
  </action>
  <action name="actionPlay_File">
   <property name="text">
    <string>Play File</string>
   </property>
  </action>
  <action name="actionMicrophone">
   <property name="text">
    <string>Two-Way Microphone</string>
   </property>
  </action>
  <action name="actionServer_Start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Server Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
